src/main/java/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterAction.java

```diff
@@ -213,8 +216,8 @@ private void updateEnabledState(AnActionEvent event) {
      */
     private void createPresenterModuleLinkForGin() {
         // 1. first search parent
-        PsiClass unit = packageHierarchy.findInterfaceTypeInParentPackage(
-                presenterConfigModel.getSelectedPackageRoot(), "GinModule");
+        PsiClass unit = packageHierarchy.findInterfaceTypeInPackage(
+                presenterConfigModel.getSelectedPackageRoot().getParentPackage(), "GinModule");
```

Comment:
Not sure, but I think AbstractGinModule is safer because the `bind()` methods are defined there